cli/src/main/java/com/box/l10n/mojito/cli/command/PushCommand.java

```diff
@@ -87,6 +96,16 @@ public void execute() throws CommandException {
         List<PollableTask> pollableTasks = new ArrayList<>();
 
         ArrayList<FileMatch> sourceFileMatches = commandHelper.getSourceFileMatches(commandDirectories, fileType, sourceLocale, sourcePathFilterRegex);
+
+        // filter out source files which do not contain diff changes if git diff is enabled.
```

Comment:
use a log statement instead of the comment please.

Reply:
moving to log statement